daemon/daemon_linux.go

```diff
@@ -20,7 +20,7 @@ import (
 // instead of deriving path from daemon's exec-root. This is because
 // plugin socket files are created here and they cannot exceed max
 // path length of 108 bytes.
-func getPluginExecRoot(root string) string {
+func getPluginExecRoot(_ *config.Config) string {
```

Comment:
Fun fact: if all args are unused, you can elide the names completely.
```suggestion
func getPluginExecRoot(*config.Config) string {
```

Reply:
Oh! TIL didn't know that!